WordPress/Classes/ViewRelated/Views/WPRichText/WPRichTextImage.swift

```diff
@@ -4,10 +4,12 @@ open class WPRichTextImage: UIControl, WPRichTextMediaAttachment {
 
     // MARK: Properties
 
+    private var reloadImage: (() -> Void)?
     var contentURL: URL?
     var linkURL: URL?
 
     @objc fileprivate(set) var imageView: CachedAnimatedImageView
+    private lazy var retryImageView = ReaderRetryFailedImageView.loadFromNib()
```

Comment:
I try to only unit test "public" interfaces so I didn't change `ReaderRetryFailedImageView` to `internal` just for testing purposes. I think views can sometimes be tricky because they can be `private` while still forming part of the public interface to the system. This is where I think UI tests can really fill in the gaps to provide good overall coverage.